packages/story-editor/src/elements/shape/icon.js

```diff
@@ -39,12 +39,15 @@ const Container = styled.div`
   background-color: ${({ theme }) => theme.colors.opacity.black10};
 `;
 
-const ShapePreview = styled.div`
+const ShapePreview = styled.div.attrs(({ backgroundColor }) => ({
+  // https://stackoverflow.com/a/58095676
+  // Removes console warning where class was re-generated over 200 times.
```

Comment:
Not sure how valuable that comment is. Especially since this is not the only place where we use `attrs()`.

I'd prefer a link to https://github.com/GoogleForCreators/web-stories-wp/pull/10753 and/or https://styled-components.com/docs/faqs#when-to-use-attrs